.github/workflows/asana-release.yml

```diff
@@ -25,30 +36,54 @@ jobs:
           RELEASE_URL: ${{ github.event.release.html_url }}
           RELEASE_NOTES: ${{ github.event.release.body }}
         run: |
-          cd autofill
-          npm install
-          JSON_STRING="$(node ./scripts/release/asana-create-tasks.js)"
+          JSON_STRING="$(node ./autofill/scripts/release/asana-create-tasks.js)"
           echo "ASANA_OUTPUT=$JSON_STRING" >> $GITHUB_OUTPUT
-          cd ..
 
+# ------------------------------------------------------------------------------
+# Create PR with updated autofill on Android
+# ------------------------------------------------------------------------------
+
+  update_android:
+    runs-on: ubuntu-20.04
+    outputs:
+      pull-request-url: ${{ steps.create-pr.outputs.pull-request-url }}
+    needs: create_asana_tasks
+    steps:
       - name: Checkout Android
         uses: actions/checkout@v3
         with:
           repository: duckduckgo/android
           path: android/
           ref: develop
           token: ${{ secrets.DAXMOBILE_AUTOFILL_AUTOMATION }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: current
```

Comment:
Android has no specified node version in use. 
- `npm` is used in an [automated task](https://github.com/duckduckgo/Android/blob/3c138ca90c815fa6c944108a7a0e681998fbc765/.github/workflows/update-ref-tests.yml) to update the reference tests
-  That job uses `ubuntu-latest`, which is currently Ubuntu 22.04.1
- The [Ubuntu 22.04 runner image](https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230206.1) uses Node.js v18.14.0

As such, this should be updated to `"*"` which ["will get the latest locally-cached Node.js version"](https://github.com/actions/setup-node#supported-version-syntax) and so match the same the same version as android.